config/environments/production.rb

```diff
@@ -19,7 +19,9 @@
   config.serve_static_files = ENV["RAILS_SERVE_STATIC_FILES"].present?
 
   # Compress JavaScripts and CSS.
-  config.assets.js_compressor = Uglifier.new(harmony: true)
+  if ENV["RPM_BUILD_ROOT"]
+    config.assets.js_compressor = Uglifier.new(harmony: true)
+  end
```

Comment:
@jordimassaguerpla what about this workaround ? this should ensure that it's only called during build time

Reply:
If that works, lgtm